projects/igniteui-angular/src/lib/drop-down/README.md

```diff
@@ -70,10 +70,46 @@ The ***igx-drop-down-item-group*** component can be used inside of the ***igx-dr
 
 ***NOTE:*** The ***igx-drop-down-item-group*** tag can be used for grouping of ***igx-drop-down-item*** only an will forfeit any other content passed to it. 
 
-### API Summary
+## Virtualized item list
+The `igx-drop-down` supports the use of `IgxForOf` directive for displaying very large lists of data. To use a virtualized list of items in the drop-down, follow the steps below:
+
+### Import IgxForOfModule
+```typescript
+    import { ..., IgxForOfModule } from 'igniteui-angular';
+    ...
+    @NgModule({
+        imports: [..., IgxForOfModule]
+    })
+```
+
+### Properly configure the template
+Configure the drop-down to use `*igxFor` instead of `ngFor`. Some additional configuration must be passed:
+ - scrollOrientation - should be `'vertical'`
+ - containerSize - should be set to the height that the items contain will have, as `number`. E.g. `public itemsMaxHeight = 480;`
```

Comment:
items contain**er** may be

Reply:
Thanks for catching that, it's addressed!